src/jsp/jsp_cl.c

```diff
@@ -2872,7 +1699,7 @@ jsp_do_call_stored_procedure (DB_VALUE * returnval, DB_ARG_LIST * args, const ch
 
   db_make_null (&method);
   db_make_null (&param);
-  memset (&sp_args, 0, sizeof (SP_ARGS));
+  //memset (&sp_args, 0, sizeof (SP_ARGS));
```

Comment:
why not needed ?

Reply:
SP_ARGS is now a C++ struct, and member variables are initialized in the constructor. Also, This line occurs SEGFAULT.